### ISSUE TITLE
Connot find module 'sass'

### ISSUE THREAD

**Issue by vortex314** [grapoza/vue-tree#176]
**Describe the bug**
I cannot compile the code 
**To Reproduce**
- I build the project scaffolding with vue-ui
![image](https://user-images.githubusercontent.com/5508735/100878440-7b246700-34aa-11eb-9533-48ef8a967ad7.png)
- compilation fails
![image](https://user-images.githubusercontent.com/5508735/100878488-8d9ea080-34aa-11eb-86bb-cccdcdbb5db1.png)

**Expected behavior**
Looks to be missing a module but it's there ? 
**Screenshots**
See above 
 - Browser Chrome on MacBook

**Additional context**

**Comment by vortex314** [comment]
Disclaimer :  I'm novice in Vue
```
<template>
  <div id="app">
    <img alt="Vue logo" src="./assets/logo.png" />
    <HelloWorld msg="Welcome to Your Your Vue.js App  " />
    <tree-view id="customtree-basic" :initial-model="model"></tree-view>
  </div>
</template>

<script>
import HelloWorld from "./components/HelloWorld.vue";
import TreeView from "@grapoza/vue-tree";

export default {
  name: "App",
  components: {
    HelloWorld,
    TreeView: TreeView,
  },
  data() {
    return {
      dataModel: [
        {
          id: "numberOrString",
          label: "Root Node",
          children: [
            { id: 1, label: "Child Node" },
            { id: "node2", label: "Second Child" },
          ],
        },
      ],
    };
  },
  computed: {},
};
</script>

<style>
#app {
  font-family: Avenir, Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  margin-top: 60px;
}
</style>
```

**Comment by vortex314** [comment]
Failed to compile.

./node_modules/@grapoza/vue-tree/components/TreeViewNode.vue?vue&type=script&lang=js& (./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./node_modules/@grapoza/vue-tree/components/TreeViewNode.vue?vue&type=script&lang=js&)
Module not found: Error: Can't resolve '../objectMethods' in '/Users/mg61dd/workspace/P00002-lieven-playground/src/main/resources/www/browse/node_modules/@grapoza/vue-tree/components'

**Comment by vortex314** [comment]
I manually copied the github file objectMethods to the package deployed locally, apparently incomplete somewhere.
compiles and works now ok

**Comment by grapoza** [comment]
@vortex314 Thanks for the thorough investigation. I'm betting (relatively) recent changes to the project structure mean that `objectMethods` file gets missed during packaging. It should be a trivial update I can get fixed up after work today. I'll use this issue to log the work since you were able to include all the relevant detail for me already.

**Comment by vortex314** [comment]
Thanks ! Glad to see you keep this project alive. :-)

**Comment by grapoza** [comment]
Glad to help! That merge to close this issue was for the new 3.0.0 that's in progress in the master branch, but I just released a patch (2.1.2) which you should be able to use in your project. It should work now, but let me know if it doesn't and I can look into it further.
